Add Using holistic event information in the trigger arXiv:1808.00711

dtsbourg opened the pull request:
Hi 👋!

Making this PR to add the report on work I did as an intern at LHCb over the last 6 months. It covers the use of Machine Learning methods in Upgrade Trigger, and I'm sure it'll be of interest to the HEP-ML community :)

I has been released as a public [LHCb note](https://cds.cern.ch/record/2632767) and is available on [Inspire HEP](https://inspirehep.net/record/1684792) or [arXiv](https://arxiv.org/abs/1808.00711).

Cheers,
Dylan

dtsbourg committed:
add arXiv:1808.00711

matthewfeickert commented:
@dtsbourg Thanks for the PR --- looks nice! I see that there was a revision that was submitted on 2018-08-08 (hence you're using it for the data instead of the 2nd) but was it a big enough revision that we should depart from using the date submitted to INSPIRE? There is no answer I'm looking for here, just wondering what the difference is and if you have a preference (though I guess the 8th as that's what you submitted).

dtsbourg commented:
The revision was due to a mistake on my part regarding the reference number (I had attributed it a mistaken `CERN-EP` number), but the content is equivalent. Either date is fine!

matthewfeickert commented:
Cool. I'm going to then grab your branch and push an update to roll back the data to the 2nd. Congrats on the nice note and thanks for your contribution!

matthewfeickert committed:
Change date to August 2nd

2018-08-02 was the date that the content itself was submitted, and
though minor revisions were made the original date seems to be what is
important here.
Additionally, the arXiv subject section is changed to Instrumentation
and Detectors (physics.ins-det) to match the primary label on
arXiv:1808.00711.
